MVP Module/Source/Feed Presentation/FeedPresenter.swift

```
@@ -5,6 +5,10 @@
 import Foundation
 import FeedFeature
 
+struct FeedErrorViewModel {
+    let message: String
```

The `message ` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `feedErrorView.display(FeedErrorViewModel(message: .none))` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's often easier to manage state when all changes are centralized.